service/ztests/accept-header.yaml

```
@@ -0,0 +1,22 @@
+script: |
+  source service.sh
+  zed create -q test
+  echo '{ts:0}' | zed load -use test -q -
+  for accept in "text/plain, application/json" "application/xml, text/css"; do
+    echo === $accept ===
```

If you use `//`, the output will be valid ZSON, so `runParquetBoomerangs` and `runZSONBoomerangs` in `zed_test.go` will be able to understand it.
```suggestion
    echo // $accept
```